transmart-copy/README.md

```diff
@@ -29,6 +29,7 @@ _Parameters:_
 - `-w <file>`, `--write <file>`: Write observations to TSV file `<file>`.
 - `-v`, `--vacuum-analyze`: Vacuum analyze the `observation_fact` table.
 - `-d`, `--directory`: Specifies a data directory.
+- `-ru`, `--re-upload`: Removes and loads the studies specified in the data directory.
```

Comment:
But why not reuse the `-d` parameter then?

Reply:
ah, true. So rewriting a study would be the default behavior.